tests/localintegration/suite_test.go

```diff
@@ -0,0 +1,19 @@
+package localintegration
+
+import (
+	"testing"
+
+	_ "github.com/coreeng/corectl/tests/localintegration/update"
+	_ "github.com/coreeng/corectl/tests/localintegration/version"
+	"github.com/phuslu/log"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+// For any commands which do not need write access to github apis or local config to be initialised
+func TestLocalSuite(t *testing.T) {
```

Comment:
Does it also run on CI?

Reply:
yep, through the makefile: `integration-test: build integration-test-local`